src/plugins/data-ajax/data-ajax.js

```diff
@@ -130,6 +131,8 @@ $document.on( "timerpoke.wb " + initEvent + " " + updateEvent + " ajax-fetched.w
 
 				//Resets the initial jQuery caching setting
 				jQuery.ajaxSettings.cache = jQueryCaching;
+
+				$elm.trigger( contentUpdatedEvent, { type: ajaxType, content: content } );
```

Comment:
If you want to send the ajax type, the parameter should be `ajax-type` instead of `type`